simphony_ui/openfoam_boundary_conditions.py

```diff
@@ -0,0 +1,149 @@
+from traits.api import HasStrictTraits, Float, Enum, Instance, Array
+from traitsui.api import View, Item, Tabbed, UItem, VGroup
+
+
+class BoundaryConditionModel(HasStrictTraits):
+
+    #: The type of the boundary condition
+    type = Enum('none', 'empty', 'zeroGradient', 'fixedGradient', 'fixedValue')
+
+    traits_view = View(
+        VGroup(
+            'type',
+            Item(
+                name='fixed_gradient',
+                visible_when='type == "fixedGradient"',
+            ),
+            Item(
+                name='fixed_value',
+                visible_when='type == "fixedValue"',
+            ),
+            show_border=True,
+        )
+    )
+
+
+class VelocityBoundaryConditionModel(BoundaryConditionModel):
+
+    #: A fixed value of velocity gradient for the boundary
+    fixed_gradient = Array(Float, (1, 3))
+
+    #: A fixed value of velocity for the boundary
+    fixed_value = Array(Float, (1, 3))
+
+
+class PressureBoundaryConditionModel(BoundaryConditionModel):
+
+    #: A fixed value of pressure gradient for the boundary
+    fixed_gradient = Float()
```

Comment:
Yes sorry the gradient is 3D for pressure two. I'll fix it. 

But the pressure value is a single value so the two classes are not redundant, the fixed_value is a float for the pressure and a 3D array for the velocity.